mobi/src/main/java/com/mobiperf/util/DataClient.java

```diff
@@ -0,0 +1,35 @@
+package com.mobiperf.util;
+
+import com.mobiperf.Logger;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.net.Socket;
+import java.net.UnknownHostException;
+
+public class DataClient implements Runnable {
+
+    private static String SERVER_ADDRESS="10.0.0.4";
```

Comment:
I am not happy about this.
We need to have a better way of saving the server address so that we avoid recompiling and redoing this section. I know this is the first attempt to before merging I think we should fix this